sys-kernel/bliss-initramfs: re-adding package

fearedbliss opened the pull request:
This re-adds the `bliss-initramfs` package. `bliss-initramfs` use to be in the gentoo tree but was cleaned up after I retired.

Zlogene commented on sys-kernel/bliss-initramfs/bliss-initramfs-8.0.0.ebuild:
```diff
@@ -0,0 +1,45 @@
+GITHUB_TAG="${PV}"
+
+DESCRIPTION="Boot your system's rootfs from OpenZFS/LUKS."
+HOMEPAGE="https://github.com/${GITHUB_USER}/${GITHUB_REPO}"
```
we prohibit variables embedding in the`HOMEPAGE` now (always)

fearedbliss replied:
fixing

Zlogene commented on sys-kernel/bliss-initramfs/bliss-initramfs-8.0.0.ebuild:
```diff
@@ -0,0 +1,45 @@
+GITHUB_REPO="bliss-initramfs"
+GITHUB_TAG="${PV}"
+
+DESCRIPTION="Boot your system's rootfs from OpenZFS/LUKS."
```
you do not want to have dot in the end of the `DESCRIPTION` (minor nitpick, yes)

fearedbliss replied:
fixing

Zlogene commented on sys-kernel/bliss-initramfs/bliss-initramfs-8.0.0.ebuild:
```diff
@@ -0,0 +1,45 @@
+	dodoc README.md README-MORE USAGE
+
+	# Make a symbolic link: /sbin/bliss-initramfs
+	dosym "${EPREFIX}/opt/${PN}/${executable}" "/sbin/${PN}"
```
make it relative

fearedbliss replied:
Will fix, to note this is following the `dosym` instructions for absolute paths located in the devmanual:

https://devmanual.gentoo.org/function-reference/install-functions/index.html

Zlogene commented on sys-kernel/bliss-initramfs/metadata.xml:
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="person">
+        <email>jon@xyinn.org</email>
```
just add your previously used @gentoo.org addrr

fearedbliss replied:
fixing, I didn't add fearedbliss@g.o originally since my email is not active yet ;D.

fearedbliss commented:
Fixed the above issues and also ran `repoman commit` to commit the changes. It added Package-Manager (portage/repoman versions) as usual, but it didn't add `Signed-off-by:` (Even though I was prompted for my GPG key so the commit is signed. I added the `Signed-off-by:` manually after the commit was done.

Zlogene commented on sys-kernel/bliss-initramfs/bliss-initramfs-8.0.0.ebuild:
```diff
@@ -0,0 +1,45 @@
+PYTHON_COMPAT=( python{3_6,3_7,3_8} )
+inherit python-single-r1
+
+GITHUB_USER="fearedbliss"
```
well, you pnly use it once, why create a variable? :p

fearedbliss replied:
lol it's only used once now after I removed it from the HOMEPAGE ;D, will change.

Zlogene commented on sys-kernel/bliss-initramfs/bliss-initramfs-8.0.0.ebuild:
```diff
@@ -0,0 +1,45 @@
+inherit python-single-r1
+
+GITHUB_USER="fearedbliss"
+GITHUB_REPO="bliss-initramfs"
```
use `${PN}`

Zlogene replied:
(just without GITHUB_REPO)

fearedbliss replied:
will change

Zlogene commented on sys-kernel/bliss-initramfs/bliss-initramfs-8.0.0.ebuild:
```diff
@@ -0,0 +1,45 @@
+
+GITHUB_USER="fearedbliss"
+GITHUB_REPO="bliss-initramfs"
+GITHUB_TAG="${PV}"
```
just use `${PV}`, you do not want to have a reassignement :)

fearedbliss replied:
will fix, but what re-assignment?

Zlogene commented:
This one I merged myself.
